spec/requests/api/v1/merchants_spec.rb

```diff
@@ -157,6 +157,61 @@
     expect(returned_merchant.class).to eq(Hash)
   end
 
+<<<<<<< HEAD
+  # it "returns customers_with_pending_invoices" do
+  #   create(:merchant)
+  #   create(:)
+  #   create_list(:invoices, 3 merchant: merchant)
+
+  #   get "/api/v1/merchants/:id/customers_with_pending_invoices"
+
+  #   merchant = JSON.parse(response.body)
+
+  #   expect(response).to be_success
+
+  it "returns merchants favorite customer" do
+    merchant = create(:merchant)
+    customer_one = create(:customer)
+    customer_two = create(:customer)
+    invoice_one = create(:invoice, merchant: merchant, customer: customer_one)
+    invoice_two = create(:invoice, merchant: merchant, customer: customer_two)
+    invoice_three = create(:invoice, merchant: merchant, customer: customer_one)
+    create(:transaction, result: 'success', invoice: invoice_one)
+    create(:transaction, result: 'success', invoice: invoice_two)
+    create(:transaction, result: 'success', invoice: invoice_three)
+    
+
+    get "/api/v1/merchants/#{merchant.id}/favorite_customer"
+
+    favorite_customer = JSON.parse(response.body)
+#byebug
```

Comment:
Same with these commented out byebugs. Make sure that you take these out before pushing.